File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/response/ResponseSite.java
package egovframework.base.account.ibll.response;

import java.util.ArrayList;
import java.util.List;
import egovframework.base.model.common.Site;
import egovframework.base.response.MessageResult;

public class ResponseSite extends MessageResult {

    private List<Site> sites = new ArrayList<>();
    private boolean useMultiSite = false;

    public List<Site> getSites() {
        return sites;
    }

    public void setSites(List<Site> sites) {
        this.sites = sites;
    }

    public boolean isUseMultiSite() {
        return useMultiSite;
    }

    public void setUseMultiSite(boolean useMultiSite) {
        this.useMultiSite = useMultiSite;
    }

    public ResponseSite() {
        super();
    }

    public ResponseSite(boolean success, String message) {
        super(success, message);
    }
}

File: src/SOP8/Base/Account/Base.Account.IBLL/Response/ResponseSite.cs
using System.Collections.Generic;
using Response;
using Base.Model.Common;

namespace Base.Account.IBLL.Response
{
    public class ResponseSite : MessageResult
    {
        private List<Site> m_sites = new List<Site>();
        private bool m_useMultiSite = false;

        public List<Site> Sites
        {
            get { return m_sites; }
            set { m_sites = value; }
        }

        public bool UseMultiSite
        {
            get { return m_useMultiSite; }
            set { m_useMultiSite = value; }
        }

        public ResponseSite()
            : base()
        {
        }

        public ResponseSite(bool success, string message)
            : base(success, message)
        {
        }
    }
}
